app/src/main/res/layout/activity_settings.xml

```diff
@@ -96,6 +96,22 @@
 
             </RelativeLayout>
 
+            <RelativeLayout
+                android:id="@+id/settings_double_tap_to_lock_holder"
+                style="@style/SettingsHolderSwitchStyle"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:background="@drawable/ripple_background">
+
+                <org.fossify.commons.views.MyMaterialSwitch
+                    android:id="@+id/settings_double_tap_to_lock"
+                    style="@style/SettingsSwitchStyle"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:text="@string/double_tap_to_lock" />
+
+            </RelativeLayout>
```

Comment:
It shouldn't be before the language setting. You should rather add it at the end of `General` section.

Reply:
Moved to the end of `General` section.
![Moved to end of General section](https://github.com/user-attachments/assets/5167d696-1157-4f1f-afb6-c1ebb71150bd)